lib/src/settings.dart

```diff
@@ -213,8 +213,10 @@ class _ValueChangeObserverState<T> extends State<ValueChangeObserver<T>> {
 
   @override
   void dispose() {
+    if (mounted == false) {
```

Comment:
This can be written as `if (!mounted)`

Reply:
Yes, it should be if (!mounted). I just copied the code from the comment above. I'm sorry. It's solved now.